controllers/submission.go

```diff
@@ -141,12 +113,12 @@ func store(ctx context.Context, key *datastore.Key, code, language string) (mode
 	o := model.StoredObject{}
 
 	// We'll be storing code, so check what name the file should have in GCS.
-	fn, ok := fileNames[language]
+	fn, ok := util.FileNames[language]
 	if !ok {
 		return o, errors.New("language unknown")
 	}
 
-	submissionBucket := "coduno"
+	submissionBucket := util.SubmissionBucket
```

Comment:
This is inconsistent. Two lines later, `submissionBucket` is changed for development again. If you move the bucket names, you should move all the logic with it.